test/helpers/kubeobjects/deployment/wait.go

```diff
@@ -34,5 +35,5 @@ func WaitFor(name string, namespace string) features.Func {
 }
 
 func WaitUntilReady(resource *resources.Resources, deployment *appsv1.Deployment) error {
-	return wait.For(conditions.New(resource).DeploymentConditionMatch(deployment, appsv1.DeploymentAvailable, corev1.ConditionTrue))
+	return wait.For(conditions.New(resource).DeploymentConditionMatch(deployment, appsv1.DeploymentAvailable, corev1.ConditionTrue), wait.WithTimeout(time.Minute*15))
```

Comment:
```suggestion
	return wait.For(conditions.New(resource).DeploymentConditionMatch(deployment, appsv1.DeploymentAvailable, corev1.ConditionTrue), wait.WithTimeout(time.Minute*15))
```
Better to extract to constant and use more natural format `<Number> * time.Minute`

Reply:
Perfect example:

https://github.com/dynatrace/dynatrace-operator/blob/2bf21151d19357a0575de5d1bb1a2b42f5f4ff6d/src/controllers/dynakube/version/healthconfig.go#L14-L15